src/Sulu/Bundle/ContactBundle/Controller/ContactController.php

```diff
@@ -476,8 +482,11 @@ public function cgetAction(Request $request)
 
             $restHelper->initializeListBuilder($listBuilder, $this->getFieldDescriptors());
 
+            $listResponse = $listBuilder->execute();
+            $listResponse = $this->getContactManager()->addAvatars($listResponse, $locale);
```

Comment:
I don't think that letting the `ContactManager` modify this array is a good idea... The `ContactManager` responsibility should solely be to handle `Contact` entities, not arrays. In general it is IMO not a good idea to pass arrays around like this. I would prefer the way you had before (then you could also remove this and the `addThumbnails` from the `MediaManager`), but just load all images at once.